addons/web_tour/static/src/scss/keyframes.scss

```diff
@@ -47,3 +47,36 @@
         transform: translate(0, 3px);
     }
 }
+
+$o-tip-border-width: 2px; // from tip.scss
+$o-tip-border-animated-width: 8px;
+$o-tip-border-width-correction: $o-tip-border-animated-width - $o-tip-border-width;
```

Comment:
You are forced to do that because you are animating a border-width, which is very inefficient as it forces a whole document paint refresh. Animate something efficient which does not impact the tip layout: I think animating the box-shadow (which can be multiple) works on all supported browser. If not there are other solution to make it efficient: like creating a new element to simulate that border and animate it via transform scale, which is efficient too.